go/oasis-test-runner/scenario/e2e/basic.go

```diff
@@ -117,7 +117,7 @@ func (sc *basicImpl) Fixture() (*oasis.NetworkFixture, error) {
 					GroupSize:         1,
 					MaxBatchSize:      1,
 					MaxBatchSizeBytes: 1000,
-					BatchFlushTimeout: 10 * time.Second,
+					BatchFlushTimeout: 1 * time.Second,
```

Comment:
Buildkite can be kind of slow is this ok?

Reply:
Yeah because this is not a timeout in the failing sense but it is how long the scheduler will wait before flushing a batch. A longer interval will just make tests wait longer for no good reason.